devlog/_plan/260906_stateful_task_guidance/010_raw_boundary.md

```
@@ -0,0 +1,38 @@
+# Align the stateful raw conversation boundary
+
+## Exact diff map
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)